### PR TITLE
Update Accepted PRs

### PR DESCRIPTION
**Pull request by yannizhou05**
Updated the status for all the accepted PRs so far.

### CODE REVIEW

File: pr-data.csv [hunk 1]
```diff
@@ -1739,7 +1739,7 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar
...
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core,org.apache.nifi.controller.TestFlowController.testSynchronizeFlowWhenExistingMissingComponentsAreDifferent,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core,org.apache.nifi.controller.TestFlowController.testSynchronizeFlowWhenProposedMissingComponentsAreDifferent,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server,org.apache.nifi.distributed.cache.server.map.DistributedMapCacheTest.testSubMap,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server,org.apache.nifi.distributed.cache.server.map.DistributedMapCacheTest.testSubMap,ID,Accpeted,https://github.com/apache/nifi/pull/6486,
```
Comment (darko-marinov) [inline pr-data.csv]: Please correct misspelled `Accpeted`.

**Review by darko-marinov — CHANGES_REQUESTED** [review]
Please correct the misspelling so that the PR passes through the checker.

File: pr-data.csv (hunk 1 above)
Comment (yannizhou05) [reply pr-data.csv]: I am sorry... I have updated the typo.

File: pr-data.csv [hunk 2]
```diff
@@ -1739,7 +1739,7 @@ https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar
...
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core,org.apache.nifi.controller.TestFlowController.testSynchronizeFlowWhenExistingMissingComponentsAreDifferent,ID,,,
 https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core,org.apache.nifi.controller.TestFlowController.testSynchronizeFlowWhenProposedMissingComponentsAreDifferent,ID,,,
-https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server,org.apache.nifi.distributed.cache.server.map.DistributedMapCacheTest.testSubMap,ID,,,
+https://github.com/apache/nifi,2bd752d868a8f3e36113b078bb576cf054e945e8,nifi-nar-bundles/nifi-standard-services/nifi-distributed-cache-services-bundle/nifi-distributed-cache-server,org.apache.nifi.distributed.cache.server.map.DistributedMapCacheTest.testSubMap,ID,Accepted,https://github.com/apache/nifi/pull/6486,
```
Comment (xieyt2000) [inline pr-data.csv:1742]: The PR https://github.com/snowflakedb/snowflake-jdbc/pull/1164 should be counted as `InspiredAFix`, not `Accepted`.

**Review by xieyt2000 — CHANGES_REQUESTED** [review]

File: pr-data.csv (hunk 2 above)
Comment (yannizhou05) [reply pr-data.csv:1742]: @xieyt2000 Can I know why? It seems like the developer directly merged my PR as `sfc-gh-igarish merged 2 commits into snowflakedb:master from yannizhou05:master yesterday`.

Comment (xieyt2000) [reply pr-data.csv:1742]: Sorry I pasted the wrong link. Should be https://github.com/apache/nifi/pull/6486 (The PR in the line I commented)

Comment (yannizhou05) [reply pr-data.csv:1742]: The developer mentioned `fixed the checkstyle issue and merged`, and the codes merged were exactly the same as my code. I thought InspiredAFix meant the developer was inspired to fix tests by their own codes?

Comment (xieyt2000) [reply pr-data.csv:1742]: You're right, it seems that it's a common practice in NiFi to close and merge by committing.
@darko-marinov, should we count that as `Accepted` or `InspiredAFix`?

Comment (darko-marinov) [reply pr-data.csv:1742]: Fine to count as `Accepted` when it's exactly the same code, even if it was not `Merged` via the GitHub interface but committed+pushed to the repo in another way.

**Review by xieyt2000 — APPROVED** [review]
LGTM

**Pull request merged**